docs/getstarted/settings.md

```diff
@@ -76,7 +78,8 @@ In the following example, the source control settings are focused by selecting *
 
 ![Settings editor with the Source Control section of the table of contents selected](images/settings/settings-groups.png)
 
-> **Note**: VS Code extensions can also add their own custom settings, and those settings are visible under an **Extensions** section.
+> [!NOTE]
```

Comment:
Does this have some special rendering? `[...]` is a special kind of link is markdown so this could maybe cause problems later.

Reply:
Yes, we added support for alert-style blockquotes on the site. The website renders these in the same way as GH.com does:

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

**Website:**
![image](https://github.com/user-attachments/assets/fcc4a9ce-247d-4de1-820d-586ee8911eed)

**GH.com:**
![image](https://github.com/user-attachments/assets/e202beeb-18db-424e-a534-232cf0360e64)